src/components/RecentSnippetItem.jsx

```diff
@@ -20,7 +20,7 @@ const RecentSnippetItem = ({ snippet }) => {
       <div className="recent-snippet-meta">
         <div>
           <Link to={`${snippet.get('id')}`} className="recent-snippet-meta-title">{title}</Link>
-          <SnippetTags className="recent-snippet-meta-tags" snippet={snippet} />
+          <SnippetTags className="recent-snippet" snippet={snippet} />
```

Comment:
What is this for?

Reply:
the `className` was changed in prev pr by mistake. `SnippetTags` component styles expect to have 'recent-snippet' as a base for classes inside that component